### PR TITLE
CIV 3852 CUI Welsh UI Translations 11

### PR DESCRIPTION
**Pull request by hmcts-version1-domini**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-3852

### Change description ###
The scope of this ticket is to develop Welsh content translation of the following screens: 

- "You chose not to try free mediation"
- "I do not agree to free mediation"

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by TatsyMaslova — APPROVED** [review]

File: src/main/views/features/mediation/i-dont-want-free-mediation.njk [hunk 1]
```diff
@@ -17,15 +18,13 @@
...
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-          <h2 class="govuk-heading-m">{{ t('I do not agree to free mediation') }}</h2>
-          {{govukHint({text: "You have chosen not to try free mediation. Please tell us why:"})}}
+          <h2 class="govuk-heading-m">{{ t('PAGES.I_DON_T_WANT_FREE_MEDIATION.DO_NOT_AGREE_TO_FREE_MEDIATION') }}</h2>
+          {% set ttext = t('PAGES.I_DON_T_WANT_FREE_MEDIATION.YOU_HAVE_CHOSEN_TELL_US_WHY') %}
+          {{govukHint({text: ttext})}}
           <form method="post" novalidate>
             {{ csrfProtection(csrf) }}
             {% set ifOther %}
```
Comment (commitBlob) [inline src/main/views/features/mediation/i-dont-want-free-mediation.njk:30]: I can't comment on line 37, can we change translation to key please

File: src/main/views/features/mediation/mediation-disagreement.njk [hunk 2]
```diff
@@ -1,63 +1,63 @@
...
 
-          <h2 class="govuk-heading-m">{{ t('Will you change your decision and try free mediation?') }}</h2>
-          <p class="govuk-body">{{ t('If you choose not to try mediation this cannot be changed once your response is submitted.') }}</p>
+          <h2 class="govuk-heading-m">{{ t('PAGES.MEDIATION_DISAGREEMENT.WILL_YOU_CHANGE_YOUR_DECISION') }}</h2>
+          <p class="govuk-body">{{ t('PAGES.MEDIATION_DISAGREEMENT.IF_YOU_CHOOSE_NOT') }}</p>
 
           <form method="post" novalidate>
             {{ csrfProtection(csrf) }}
-            {{ yesNoRadioButton(t, 'option', 'option', '', form, '', {yesHtml:yesHtml, classes: 'govuk-radios'}) }}
+            {{ yesNoRadioButton(t, 'option', 'option', '', form, '', {classes: 'govuk-radios', yes: 'COMMON.VARIATION.YES', no: 'COMMON.VARIATION.NO'}) }}
```
Comment (commitBlob) [inline src/main/views/features/mediation/mediation-disagreement.njk:57]: seems like you don't use yesHtml anymore, can we remove set block then on the line 15?

File: src/main/views/features/mediation/i-dont-want-free-mediation.njk (hunk 1 above)
Comment (hmcts-version1-domini) [reply src/main/views/features/mediation/i-dont-want-free-mediation.njk:30]: Done. Thanks!

File: src/test/unit/routes/features/response/responseDeadline/agreedResponseDeadlineController.test.ts [hunk 3]
```diff
@@ -132,7 +132,7 @@ describe('Agreed response date', () => {
         .send('day=13')
         .expect((res) => {
           expect(res.status).toBe(200);
-          expect(res.text).toContain(TestMessages.DATE_NOT_MORE_THAN_28_DAYS);
+          expect(res.text).toContain('Agreed response date cannot be more than 28 days after the original response date');
```
Comment (GorilaNaranja) [inline src/test/unit/routes/features/response/responseDeadline/agreedResponseDeadlineController.test.ts:135]: We can keep using "errorMessageTestConstants.ts" constants in tests, it is a util different than "errorMessageConstants.ts"

**Review by commitBlob — APPROVED** [review]

**Pull request closed without merging**